frontend/src/metabase/parameters/components/ParameterSettings/ParameterSettings.tsx

```diff
@@ -36,6 +38,22 @@ export interface ParameterSettingsProps {
   embeddedParameterVisibility: EmbeddingParameterVisibility | null;
 }
 
+type SectionOption = {
+  sectionId: string;
+  type: string;
+  name: string;
+  operator: string;
+  menuName?: string;
+  sidebarMenuName?: string;
+  combinedName?: string | undefined;
+};
+
+const parameterSections = getDashboardParameterSections();
```

Comment:
converting getDashboardParameterSections to typesript makes types way more complicated